infrastructure/create-env-file.js

```diff
@@ -0,0 +1,57 @@
+// reads the manifest.json and the stack template file from the cdk.out directory,
```

Comment:
Mention `manifest.json` here but I don't see it actually pulled in anywhere.

If you do use that you can use the `CloudAssembly` class CDK provides: https://docs.aws.amazon.com/cdk/api/v2/docs/@aws-cdk_cx-api.CloudAssembly.html

```
const assembly = new CloudAssembly(<path/to/cdk.out>)

// Do something with each stack
assembly.stacks.forEach(...)

// Get a specific stack
const needleStack = assembly.getStackByName(...)

// Read the entire stack template
const stackTemplate = fs.readFileSync(needleStack. templateFullPath)

// Get the stack name
const stackName = needleStack.stackName
```

This would also allow for iterating over all stacks defined in a service and pull outputs from all/some.

Reply:
I just learned `cfn.describeStacks` uses AWS CloudFormation API to get the details of the specified stack. Nothing local.